devops/makeReactBucket.json

```diff
@@ -0,0 +1,69 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Parameters": {
+    "BucketName": {
```

Comment:
we set this with `--parameter-overrides BucketName=<<BUCKET_NAME>>` and get it with `{ "Ref": "BucketName" }` in the template below